docs/installing.rst

```diff
@@ -20,3 +20,14 @@ To install from source:
 .. code-block:: bash
 
   $ python setup.py develop
+
+Optional packages
+-----------------
+
+Appium
+~~~~~~
+To install pytest-selenium with `appium <https://appium.io/>`_ support:
```

Comment:
nit: capitalise Appium for consistency

Reply:
I'll fix that, and another spelling error in the next PR (deprecating PhantomJS) :)